src/main/java/uk/gov/hmcts/juror/api/moj/service/report/YieldPerformanceReportServiceImpl.java

```diff
@@ -0,0 +1,186 @@
+        int balance = confirmed - requested;
+        double difference = requested == 0 ? 0 : round((double)balance / requested * 100);
+        response.getTableData().getData().add(
+            YieldPerformanceReportResponse.TableData.YieldData.builder()
+            .courtLocation(result.get(1) + " (" + result.get(0) + ")")
+            .requested(requested)
+            .confirmed(confirmed)
+            .comments(poolCommentsMap.get(result.get(0)) != null
+                ? poolCommentsMap.get(result.get(0)).toString()
+                : "")
+            .balance(balance)
+            .difference(difference)
+            .build());
+    }
+
+    private YieldPerformanceReportResponse setupResponseHeaders(
+        YieldPerformanceReportRequest yieldPerformanceReportRequest) {
+
+        log.info("User {} viewing yield performance report",
+            SecurityUtil.getActiveLogin());
+
+        if (!yieldPerformanceReportRequest.isAllCourts()
+            && (yieldPerformanceReportRequest.getCourtLocCodes() == null
+            || yieldPerformanceReportRequest.getCourtLocCodes().isEmpty())) {
+            throw new MojException.BadRequest("No court locations provided", null);
+        }
+
+        YieldPerformanceReportResponse response = new YieldPerformanceReportResponse();
+
+        response.setHeadings(getSearchByCourts(
+            yieldPerformanceReportRequest.getFromDate(),
+            yieldPerformanceReportRequest.getToDate()));
+
+        return response;
+    }
+
+
+    private Map<String, AbstractReportResponse.DataTypeValue>
+        getSearchByCourts(LocalDate reportFrom, LocalDate reportTo) {
+
+        return Map.of(
+            "date_from", AbstractReportResponse.DataTypeValue.builder()
+                .displayName(ReportHeading.DATE_FROM.getDisplayName())
+                .dataType(ReportHeading.DATE_FROM.getDataType())
+                .value(reportFrom.format(DateTimeFormatter.ISO_LOCAL_DATE))
+                .build(),
+            "date_to", AbstractReportResponse.DataTypeValue.builder()
+                .displayName(ReportHeading.DATE_TO.getDisplayName())
+                .dataType(ReportHeading.DATE_TO.getDataType())
+                .value(reportTo.format(DateTimeFormatter.ISO_LOCAL_DATE))
+                .build(),
+            "report_created", AbstractReportResponse.DataTypeValue.builder()
+                .displayName(ReportHeading.REPORT_CREATED.getDisplayName())
+                .dataType(ReportHeading.REPORT_CREATED.getDataType())
+                .value(LocalDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE))
+                .build(),
+            "time_created", AbstractReportResponse.DataTypeValue.builder()
+                .displayName(ReportHeading.TIME_CREATED.getDisplayName())
+                .dataType(ReportHeading.TIME_CREATED.getDataType())
+                .value(LocalDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME))
+                .build()
```

Comment:
Would be good to combine these two into one header such as "report_created_at" and return  LocalDateTime.now() as this will contain both date and time elements. 

Plus means we do not have misalignment across the two fields

Reply:
This is a possible future enhancement - will impact FE as well

Reply:
it will be good to standardise as all other reports use a single report_created header.

Reply:
updated to be same as standard report now